vitess/scripts/start.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+source ./utils.sh
```

Comment:
please prefix the present working directory, else it assumes that I need to cd into the scripts directory

Reply:
done!